framework_tvb/tvb/core/entities/file/hdf5_storage_manager.py

```diff
@@ -35,18 +35,21 @@
 .. moduleauthor:: Calin Pavel <calin.pavel@codemart.ro>
 """
 
-import os
 import copy
+import os
 import threading
+from datetime import datetime
+
 import h5py as hdf5
 import numpy as numpy
 import tvb.core.utils as utils
-from datetime import datetime
 from tvb.basic.logger.builder import get_logger
 from tvb.basic.profile import TvbProfile
 from tvb.core.entities.file.exceptions import FileStructureException, MissingDataSetException
 from tvb.core.entities.file.exceptions import IncompatibleFileManagerException, MissingDataFileException
+from tvb.core.entities.file.files_helper import FilesHelper
 from tvb.core.entities.transient.structure_entities import GenericMetaData
+from tvb.core.services.data_encryption_handler import encryption_handler
```

Comment:
This import might show that `encryption_handler.py` should stay here under `tvb.core.entities.file` (and not under `services` as I wrongly suggested before as a valid alternative).